ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-cpou-upgrade-from-4.14.yaml

```diff
@@ -18,7 +18,7 @@ base_images:
   tests-private-preupg:
     name: tests-private
     namespace: ci
-    tag: "4.16"
+    tag: "4.14"
```

Comment:
I think this change is unexpected.

Reply:
Ignore this comment, the cpou upgrade use different test steps than the chain upgrade.